pkgs/development/python-modules/k5test/fix-paths.patch

```diff
@@ -26,97 +26,100 @@ index e289fac..e2f486a 100644
+             ("kadmin_local", "kadmin.local", "/usr/sbin/kadmin.local"),
+             ("kadmind", "kadmind", "/usr/sbin/kadmind"),
+             ("kprop", "kprop", "/usr/sbin/kprop"),
+-            ("_kinit", "kinit", "/usr/bin/kinit"),
+-            ("_klist", "klist", "/usr/bin/klist"),
++            ("_kinit", "kinit", "@krb5FullDev@/bin/kinit"),
++            ("_klist", "klist", "@krb5FullDev@/bin/klist"),
          ]
  
      @property
-@@ -585,12 +585,12 @@ class HeimdalRealm(K5Realm):
+@@ -628,12 +628,12 @@ class HeimdalRealm(K5Realm):
  
          return [
-             ('krb5kdc', 'kdc', os.path.join(base, 'kdc')),
--            ('kadmin', 'kadmin', '/usr/bin/kadmin'),
--            ('kadmin_local', 'kadmin', '/usr/bin/kadmin'),
-+            ('kadmin', 'kadmin', '@krb5Full@/bin/kadmin'),
-+            ('kadmin_local', 'kadmin', '@krb5Full@/bin/kadmin'),
-             ('kadmind', 'kadmind', os.path.join(base, 'kadmind')),
--            ('_kinit', 'kinit', '/usr/bin/kinit'),
--            ('_klist', 'klist', '/usr/bin/klist'),
--            ('_ktutil', 'ktutil', '/usr/bin/ktutil'),
-+            ('_kinit', 'kinit', '@krb5Full@/bin/kinit'),
-+            ('_klist', 'klist', '@krb5Full@/bin/klist'),
-+            ('_ktutil', 'ktutil', '@krb5Full@/bin/ktutil'),
+             ("krb5kdc", "kdc", os.path.join(base, "kdc")),
+-            ("kadmin", "kadmin", "/usr/bin/kadmin"),
+-            ("kadmin_local", "kadmin", "/usr/bin/kadmin"),
++            ("kadmin", "kadmin", "@krb5FullDev@/bin/kadmin"),
++            ("kadmin_local", "kadmin", "@krb5FullDev@/bin/kadmin"),
+             ("kadmind", "kadmind", os.path.join(base, "kadmind")),
+-            ("_kinit", "kinit", "/usr/bin/kinit"),
+-            ("_klist", "klist", "/usr/bin/klist"),
+-            ("_ktutil", "ktutil", "/usr/bin/ktutil"),
++            ("_kinit", "kinit", "@krb5FullDev@/bin/kinit"),
++            ("_klist", "klist", "@krb5FullDev@/bin/klist"),
++            ("_ktutil", "ktutil", "@krb5FullDev@/bin/ktutil"),
          ]
  
      @property
 diff --git a/k5test/unit.py b/k5test/unit.py
-index 3c64b9e..59da1ab 100644
+index caa8315..5bcf0bf 100644
 --- a/k5test/unit.py
 +++ b/k5test/unit.py
-@@ -38,7 +38,7 @@ _KRB_VERSION = None
+@@ -39,7 +39,7 @@ _KRB_VERSION = None
  def krb_minversion_test(target_version, problem, provider=None):
      global _KRB_VERSION
      if _KRB_VERSION is None:
 -        _KRB_VERSION = _utils.get_output("krb5-config --version")
-+        _KRB_VERSION = _utils.get_output("@krb5FullDev@/bin/krb5-config --version")
-         _KRB_VERSION = _KRB_VERSION.split(' ')[-1].split('.')
++        _KRB_VERSION = _utils.get_output("@krb5FullDev@--version")
+         _KRB_VERSION = _KRB_VERSION.split(" ")[-1].split(".")
  
      def make_ext_test(func):
+diff --git a/setup.py b/setup.py
+index e5909f6..0132c9a 100755
+--- a/setup.py
++++ b/setup.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python
++#!@krb5FullDev@/bin/env python
```

Comment:
That should be patched with patchShebangs

Reply:
I think this was done by mistake, I removed this patching. I think `patchShebangs` is not needed for setup.py as it is run by python explicitly.

Do you agree? If not, I can patch it, sure.

Reply:
if it works, it works